src/3/world1.md

```diff
@@ -212,69 +213,69 @@ She was able to clearly recall it.
+It was obvious that at that time heaven lord Alfreyja had lost his mind.
+Whether it was brainwashing or confusion.
 Reiren couldn't possibly know details behind it.
-But she had idea of who could be a culprit.
+But she had an idea about who could be the culprit.
 
 "..."
 
-Her long azure hair spread throughout a water like a stream.
-Putting the hand on her <span title="凹凸の控えめな胸元 - I'm not sure why author needs to put such accent here">moderate</span> chest.
-Elven shrine maiden clenched her teeth.
+Her long azure hair spread throughout the water like a stream.
+Putting her hand on her <span title="凹凸の控えめな胸元 - I'm not sure why author needs to put such accent here">moderate</span> chest.
+The elven shrine maiden clenched her teeth.
 
 _TL Note:_ `深い森のなか奥歯` I don't get this one
 
 "...Patience. I have to accompany these humans for the sake of vengeance for Alfreyja-dono.
 I already resolved to make this detestable beast to realize the gravity of his crime."
 
-The one who caused heaven lord Alfreyja to change.
+The one who caused the heaven lord Alfreyja to change.
 It was highly likely that the culprit behind it was the [Fang King] Rath=IE.
 And it was a duty, bestowed upon Reiren, to ascertain it with her own eyes.
 
-"That's right. It is a honourable mission for foreign gods.
-So accompanying humans, so to speak, is a honourable duty to become elves representative...
-Right, if I'd think about it like that..."
+"That's right. It is a honourable mission for the foreign gods.
+So accompanying humans, so to speak, is a honourable duty to become the elves' representative...
+Right, if I think about it like that..."
 
 _TL Note:_ Talk about escaping reality...
 
 "Hey, hey?"
 
 "Hh-yaa!? W-what is it!"
 
-It almost made her to cream, and then Reiren turned over.
+It almost made her to scream, and then Reiren turned over.
 Cold hand, wet in water, which reached her back now touched her cheek.
-Of course she raised her voice because of sudden touch from cold hand, rather than being surprised.
+Of course she raised her voice because of the sudden touch from the cold hand, rather than being surprised.
 
-"...Mu. As I remember you're Rinne."
+"...Mu. As I remember thou are Rinne."
 
 "Yup."
 
 Reiren wondered when she came to this spring.
 Before her was a golden haired girl, her body up to chest being under water.
-On the bank of the spring could be seen her clothes, it would seem that this girl swam all the way here.
+Her clothes could be seen on the bank of the spring; it would seem that this girl swam all the way here.
 
 "It feels so good.
-This water, it is spring water, right?
+This water is spring water, right?
 Cold, but so pretty.
 And there is even small fish swimming around."
 
 "U-umu.
 It is filtered water from within underground after all.
-For every living creature in the forest this is a watering place, with abundance of clear water....
+For every living creature in the forest this is a basin, with an abundance of clear water....
```

Comment:
`basin` - please check original word, I think it was more like a watering place as if the place where animals gather to drink and wash

Reply:
I understood... tried to find a better word for "watering place" but could not find one... 
I wanted to go with the meaning 2b here: https://www.merriam-webster.com/dictionary/basin